### DISCUSSION TITLE
The proper way to silence notifications when app is in foreground

### DISCUSSION THREAD

**Discussion by Too-Many-Bytes** [mirzemehdi/KMPNotifier#99] [Q&A] [answered]
Hello! Could you, please, tell if it is possible to block getting notifications while app is in foreground and show them, when app is closed?

**Comment by mirzemehdi** [comment, marked as answer]
There is this `showPushNotification` field when initializing the library. Not sure if that's what you are looking for @Too-Many-Bytes 



```
/**
        * By default showPushNotification value is true.
        * When set showPushNotification to false foreground push  notification will not be shown to user.
        * You can still get notification content using #onPushNotification listener method.
        */

NotifierManager.initialize(
           configuration = NotificationPlatformConfiguration.Android(
               notificationIconResId = R.drawable.ic_launcher_foreground,
               showPushNotification = true,
           )
       )
       
```

**Reply by blakelee** [reply]
I was looking for this as well. I want to be able to disable push notifications when the app is in the foreground so that I can show custom in-app notifications. It looks like once NotifierManager is initialized you can't change the configuration which makes this difficult to do.
